homeassistant/components/deluge/const.py

```diff
@@ -1,17 +1,37 @@
 """Constants for the Deluge integration."""
 
+import enum
 import logging
 from typing import Final
 
 CONF_WEB_PORT = "web_port"
 CURRENT_STATUS = "current_status"
-DATA_KEYS = ["upload_rate", "download_rate", "dht_upload_rate", "dht_download_rate"]
 DEFAULT_NAME = "Deluge"
 DEFAULT_RPC_PORT = 58846
 DEFAULT_WEB_PORT = 8112
 DOMAIN: Final = "deluge"
 DOWNLOAD_SPEED = "download_speed"
 
 LOGGER = logging.getLogger(__package__)
-
+DHT_DOWNLOAD_RATE = "dht_download_rate"
+DHT_UPLOAD_RATE = "dht_upload_rate"
```

Comment:
These are duplicated in the enum now. You can use a StrEnum to make sure the values remain strings. Please only keep one of them. Either the enum or the separate constant.

Reply:
so everyone (including myself) is getting confused on these. The keys in const.py were for the translation keys and the sensors, while the other ones were for the Deluge API response

I have created another enum (and converted them both to StrEnum) to explicitly call out these are just for the sensor / translation keys, even if they are named very similarly